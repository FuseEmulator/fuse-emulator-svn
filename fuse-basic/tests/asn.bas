10 PRINT ASN -1
20 PRINT ASN 0
30 PRINT ASN 1
40 PRINT ASN 0.575
50 PRINT ASN -0.251
60 PRINT SIN ASN 0.507
