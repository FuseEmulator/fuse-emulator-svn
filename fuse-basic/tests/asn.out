-1.5708
0
1.5708
0.612604
-0.253713
0.507
0 OK
